rs/registry/admin/BUILD.bazel

```diff
@@ -134,13 +133,16 @@ rust_test(
     deps = DEPENDENCIES,
 )
 
-rust_test(
+test_using_pocket_ic_server(
     name = "integration_tests",
     srcs = glob(["tests/**/*.rs"]),
     aliases = ALIASES,
     data = DEV_DATA,
     env = DEV_ENV,
+    # Run the tests using the mainnet pocket-ic server on PRs but run against the HEAD version every night:
+    extra_HEAD_tags = ["nns_tests_nightly"],
     flaky = False,
+    macro = rust_test,
     proc_macro_deps = MACRO_DEPENDENCIES,
     deps = DEPENDENCIES + DEV_DEPENDENCIES,
 )
```

Comment:
@dfinity/nns-team I'm not sure with what version of the pocket-ic server we should run the ic-admin integration tests. Should we run it using the HEAD version on PRs or just with the mainnet version and only use HEAD on nightly?

Reply:
I don't think that it actually matters, b/c ic-admin interacts with registry and governance, and those don't need a particular replica version.